app/(candidate)/dashboard/components/DashboardHeader.js

```diff
@@ -1,22 +1,27 @@
 'use client'
-import { useCampaign } from '@shared/hooks/useCampaign'
 import { CampaignCountdown } from 'app/(candidate)/dashboard/components/CampaignCountdown'
 import { VoterContactsCount } from 'app/(candidate)/dashboard/components/VoterContactsCount'
 import { RemainingTasks } from 'app/(candidate)/dashboard/components/RemainingTasks'
 import { CampaignProgress } from 'app/(candidate)/dashboard/components/CampaignProgress'
 
-export const DashboardHeader = () => {
-  const [campaign] = useCampaign()
-  const { pathToVictory: p2vObject, details: campaignDetails } = campaign || {}
+export const DashboardHeader = ({ campaign, tasks = [] }) => {
+  const {
+    pathToVictory: p2vObject,
+    details: campaignDetails,
+    completedTaskIds,
+  } = campaign || {}
   const pathToVictory = p2vObject?.data || {}
   const { electionDate: electionDateStr } = campaignDetails || {}
   const electionDate = new Date(electionDateStr)
 
+  // TODO: this is wrong! need to calculate completed tasks for current week
+  const numOfRemainingTasks = tasks.length - (completedTaskIds?.length || 0)
+
   return (
     <section className="mb-6">
       <CampaignCountdown electionDate={electionDate} />
       <VoterContactsCount {...{ pathToVictory }} />
-      <RemainingTasks />
+      <RemainingTasks numOfRemainingTasks={numOfRemainingTasks} />
```

Comment:
I'm not seeing any change to this component to consume this change to props? 🤔 

Maybe also, just pass the remaining tasks and then the prop can just be `tasks` and then a `.length` for the output internally in the component? Seems a bit cleaner to me that way.